packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -1271,28 +1916,84 @@ class RoundSliderThumbShape extends SliderComponentShape {
     return Size.fromRadius(isEnabled ? enabledThumbRadius : _disabledThumbRadius);
   }
 
+  @override
+  void paint(
+      PaintingContext context,
+      Offset center, {
+        Animation<double> activationAnimation,
+        Animation<double> enableAnimation,
+        bool isDiscrete,
+        TextPainter labelPainter,
+        RenderBox parentBox,
+        SliderThemeData sliderTheme,
+        TextDirection textDirection,
+        double value,
+      }) {
+    final Canvas canvas = context.canvas;
+    final Tween<double> radiusTween = Tween<double>(
+      begin: _disabledThumbRadius,
+      end: enabledThumbRadius,
+    );
+    final ColorTween colorTween = ColorTween(
+      begin: sliderTheme.disabledThumbColor,
+      end: sliderTheme.thumbColor,
+    );
+    canvas.drawCircle(
+      center,
+      radiusTween.evaluate(enableAnimation),
+      Paint()..color = colorTween.evaluate(enableAnimation),
+    );
+  }
+}
+
+/// This is the default shape of a [RangeSlider]'s thumbs.
+///
+/// See also:
+///
+///  * [RangeSlider], which includes thumbs defined by this shape.
+///  * [SliderTheme], which can be used to configure the thumb shapes of all
+///    range sliders in a widget subtree.
+class RoundRangeSliderThumbShape extends RangeSliderThumbShape {
+  /// Create a slider thumb that draws a circle.
+  const RoundRangeSliderThumbShape({
+    this.enabledThumbRadius = 10.0,
+    this.disabledThumbRadius,
+  });
+
+  /// The preferred radius of the round thumb shape when the slider is enabled.
+  ///
+  /// If it is not provided, then the material default of 10 is used.
+  final double enabledThumbRadius;
+
+  /// The preferred radius of the round thumb shape when the slider is disabled.
+  ///
+  /// If no disabledRadius is provided, then it is equal to the
+  /// [enabledThumbRadius]
```

Comment:
This could probably fit on the line above (and also add a period at the end of the sentence).

Reply:
doesn't fit, added period